bigip/resource_bigip_ltm_node.go

```diff
@@ -57,14 +57,19 @@ func resourceBigipLtmNode() *schema.Resource {
 			"monitor": {
 				Type:        schema.TypeString,
 				Optional:    true,
+				Default:     "/Common/icmp",
 				Description: "Specifies the name of the monitor or monitor rule that you want to associate with the node.",
-				Computed:    true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "Test-Node",
```

Comment:
@RavinderReddyF5 this Default shouldn't be here, every user using the latest version of the provider (v0.12.4) now has this string as a description on all their nodes!

/cc @scshitole this needs to be fixed and I would suggest the release to be reverted until then.